test: add SinglelinePage tests

R894 opened the pull request:
# Description
Adds tests for single line page

NoamGaash reviewed (approved):
lgtm, 10x!

DanLeshinsky commented on src/test_pages/TimelinePage.ts:
```diff
@@ -38,8 +38,6 @@ export default class TimelinePage extends BasePage {
-    isVisible
-      ? await expect(this.route_select).toBeVisible({ timeout: 2000 })
-      : await expect(this.route_select).toBeHidden({ timeout: 2000 })
+    await this.verifySelectionVisible(this.route_select, isVisible)
```
Hi! I suggest you not to mix between the tickets. If you write tests for the single line page, you shouldn't think this time about anything except this page.

R894 replied:
I will keep this in mind in future PRs, I've changed it back to how it was before to avoid conflicts with the other tickets.
I'm open to further feedback and would be happy to make additional adjustments if needed.

Thank you!

R894 commented:
Hi @DanLeshinsky ,

Thank you for the feedback! I've addressed the concerns you raised and implemented the requested changes. Please take another look at the code, and let me know if everything looks good to you.

If you have any further suggestions or if there's anything else you'd like me to modify, please don't hesitate to let me know.

Thanks again for your time and feedback!
